docs/index.md

```diff
@@ -15,1621 +15,118 @@ parent="smn_workw_machine"
-Creating SSH key...
-Creating Digital Ocean droplet...
-To see how to connect Docker to this machine, run: docker-machine env staging
-```
-
-For convenience, `docker-machine` will use sensible defaults for choosing
-settings such as the image that the VPS is based on, but they can also be
-overridden using their respective flags (e.g. `--digitalocean-image`). This is
-useful if, for instance, you want to create a nice large instance with a lot of
-memory and CPUs (by default `docker-machine` creates a small VPS). For a full
-list of the flags/settings available and their defaults, see the output of
-`docker-machine create -h`.
-
-When the creation of a host is initiated, a unique SSH key for accessing the
-host (initially for provisioning, then directly later if the user runs the
-`docker-machine ssh` command) will be created automatically and stored in the
-client's directory in `~/.docker/machines`. After the creation of the SSH key,
-Docker will be installed on the remote machine and the daemon will be configured
-to accept remote connections over TCP using TLS for authentication. Once this
-is finished, the host is ready for connection.
-
-To prepare the Docker client to send commands to the remote server we have
-created, we can use the subshell method again:
-
-```
-$ eval "$(docker-machine env staging)"
-```
-
-From this point, the remote host behaves much like the local host we created in
-the last section. If we look at `docker-machine ls`, we'll see it is now the
-"active" host, indicated by an asterisk (`*`) in that column:
+In addition to `docker` commands, you can also use `docker-machine` commands to
+manage your machines. For example, you can use the `ls` subcommand to list the
+machines you have and their state:
 
 ```
 $ docker-machine ls
-NAME      ACTIVE   DRIVER         STATE     URL
-dev                virtualbox     Running   tcp://192.168.99.103:2376
-staging   *        digitalocean   Running   tcp://104.236.50.118:2376
+NAME              ACTIVE   DRIVER       STATE     URL
+dev                        virtualbox   Stopped
+my-docker-machine          virtualbox   Running   tcp://192.168.99.100:2376
+machine-2                  virtualbox   Running   tcp://192.168.99.106:2376
+machine-3   *              virtualbox   Running   tcp://192.168.99.107:2376
 ```
 
-To remove a host and all of its containers and images, use `docker-machine rm`:
+There are over fifteen different [subcommands for managing machines](/reference).
+         
+Of course, where Docker Machine really excels is in provisioning, managing, and
+configuring multiple Docker hosts. These hosts can be on your local network or
+on a remote, cloud provider. Further, Docker Machine makes it easy to deploy
+machines that use other Docker platform components such as Docker Compose and
+Docker Swarm.
 
-```
-$ docker-machine rm dev staging
-$ docker-machine ls
-NAME      ACTIVE   DRIVER       STATE     URL
-```
+## Choosing a provider and a base operating systems
```

Comment:
What I"d like to do is actually split these two out and then rewrite portions.  The reason I added information to this file was it literally made no sense after I split.  If the actual split is ok, we can fix the index.md after.  Holding the split prevents working on the individual sections difficult while we wait for this larger PR to  merge. WDYT?